src/http/cf.rs

```diff
@@ -12,17 +13,37 @@ use crate::http::{feature::headers, DEFAULT_HTTP_TIMEOUT_SECONDS};
 use crate::settings::global_user::GlobalUser;
 use crate::terminal::emoji;
 use crate::terminal::message::{Message, StdOut};
+
+const ENDPOINT_URL: &str = "ENDPOINT_URL";
```

Comment:
Nit: but could you rename this to something like `ENDPOINT_BASE_URL` ? Otherwise it sort of sounds like you're able to control the entire URL, not just the host / base of the URL.

Reply:
Makes sense. Updated in 54b4860.